aspnetcore/security/enforcing-ssl.md

```diff
@@ -24,16 +24,17 @@ No API can prevent a client from sending sensitive data on the first request.
 > * Not listen on HTTP.
 > * Close the connection with status code 400 (Bad Request) and not serve the request.
 
-<a name="require"></a>
-
 ## Require HTTPS
 
 ::: moniker range=">= aspnetcore-2.1"
 
-We recommend all production ASP.NET Core web apps call:
+We recommend that production ASP.NET Core web apps call:
+
+* HTTPS Redirection Middleware ([UseHttpsRedirection](/dotnet/api/microsoft.aspnetcore.builder.httpspolicybuilderextensions.usehttpsredirection)) to redirect HTTP requests to HTTPS.
```

Comment:
I was 🏃🏃🏃😅 at the time, so that's why I didn't put them in. @Rick-Anderson told me offline that it is kind'a optional, especially when under pressure to get an issue closed. I only skip putting them in when I'm 😵🏃 and the topic is already full of the old style link style.

I'm not so crazy busy today ... I'll update them on the next commit here, but just these ones that I'm adding. As I get back around on UE passes and topic overhauls (like the Distributed Cache PR that I'm turning in today), you'll see that I'm updating *all* of the links in the topic. I ❤️ that I get validation + can click them in a locally-built topic.